sbt-ci-openjdk8/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM adoptopenjdk/openjdk8:x86_64-ubuntu-jdk8u252-b09-slim
+
+ARG SBT_VERSION=1.3.10
+
+LABEL version="jdk8u252-b09-$SBT_VERSION"
```

Comment:
I suggest listing the CPU architecture type.
e.g. `LABEL version="x86_64-jdk8u252-b09-$SBT_VERSION"`

Reply:
It does not change, because this image is intended for internal use within Chatwork.